src/babel.coffee

```diff
@@ -132,7 +132,7 @@ transpile = (sourceCode, filePath, cachePath) ->
 # either generated on the fly or pulled from cache.
 loadFile = (module, filePath) ->
   sourceCode = fs.readFileSync(filePath, 'utf8')
-  unless sourceCode.startsWith('"use 6to5"') or sourceCode.startsWith("'use 6to5'")
+  unless /^("use 6to5"|'use 6to5'|"use babel"|'use babel')/.test(sourceCode)
```

Comment:
I copy and pasted that line from [atom-shell-starter](https://github.com/atom/atom-shell-starter) so I have no idea how well it fairs.